server/src/main/java/org/eclipse/lsp/cobol/core/visitor/VariableDefinitionDelegate.java

```diff
@@ -345,6 +384,13 @@ private void addError(String suggestion, Locality locality, ErrorSeverity severi
         format("Syntax error defined by %s: %s", getClass().getSimpleName(), error.toString()));
   }
 
+  private void checkRedefinesContainsValue(VariableDefinitionContext variable) {
+    if (!(variable.getRedefinesClauses().isEmpty() || variable.getValueClauses().isEmpty())) {
+      Locality valueLocality = positions.get(variable.getValueClauses().get(0).getStart());
```

Comment:
THe locality may miss, so it would be better to wrap it with `Optional.ofNullable().ifPresent()`

Reply:
Yep.